docs/src/dplyr.md

```diff
@@ -0,0 +1,293 @@
+```
+
+The columns (in order) correspond to the following: 
+
+column name | Description
+--- | ---
+`:name` | common name
+`:genus` | taxonomic rank
+`:vore` | carnivore, omnivore or herbivore?
+`:order` | taxonomic rank
+`:conservation` | the conservation status of the mammal
+`:sleep_total` | total amount of sleep, in hours
+`:sleep_rem` | rem sleep, in hours
+`:sleep_cycle` | length of sleep cycle, in hours
+`:awake` | amount of time spent awake, in hours
+`:brainwt` | brain weight in kilograms
+`:bodywt` | body weight in kilograms
+
+
+# Important DataFramesMeta.jl Verbs To Remember
+
+dplyr verbs | Description
+--- | ---
+`@select` | select columns 
+`@subset` | filter rows
+`@orderby` | re-order or arrange rows
+`@transform` | create new columns
+`@combine` | summarise values
+`groupby` | allows for group operations in the "split-apply-combine" concept
+
+DataFramesMeta.jl also provides `@rselect`, `@rsubset`, `@rorderby`, and `@rtransform` for operations which act row-wise. We will expore the distinction between column-wise and row-wise transformations later in this turorial. 
+
+# DataFramesMeta.jl Verbs In Action
+
+Two of the most basic functions are `@select` and `@subset`, which selects columns and filters rows respectively. To reference columns, use the `Symbol` of the column name, i.e. `:name` refers to the column `msleep.name`.
+
+## Selecting Columns Using `@select`
+
+Select a set of columns: the `:name` and the `:sleep_total` columns. 
+
+```@example 1
+sleepData = @select msleep :name :sleep_total
+```
+
+To select all the columns *except* a specific column, use the `Not` function for inverse selection. We preface the `Not` with `$` because it does not reference a column directly as a `Symbol`.
+
+```@example 1
+@select msleep $(Not(:name))
+```
+
+To select a range of columns by name, use the `Between` operator:
+
+```@example 1
+@select msleep $(Between(:name, :order))
+```
+
+To select all columns that start with the character string `"sl"` use regular expressions. 
+
+```@example 1
+@select msleep $(r"^sl")
+```
+
+Regular expressions are powerful, but can be difficult for new users to understand. Here are some quick tips.  
+
+1. `r"^abc"` = Starts with `"abc"`
+2. `r"abc$"` = Ends with `"abc"`
+3. `r"abc"` = Contains `"abc"` anywhere. 
+
+## Selecting Rows Using `@subset` and `@rsubset`
+
+Filter the rows for mammals that sleep a total of more than 16 hours. 
+
+```@example 1
+@subset msleep :sleep_total .>= 16
+```
+
+In the above expression, the `.>=` means we "broadcast" the `>=` comparison across the whole column. We can use a simpler syntax, `@rsubset` which automatically broadcasts all operations.
+
+```@example 1
+@rsubset msleep :sleep_total > 16
+```
+
+Subset the rows for mammals that sleep a total of more than 16 hours *and* have a body weight of greater than 1 kilogram. For this we put multiple operations on separate lines in a single block. 
+
+```@example 1
+@rsubset msleep begin 
```

Comment:
the reader is not clear if `@rsubset(msleep, :sleep_total >= 16, :bodywt >= 1)` would work or not.

Reply:
Thanks. added.